metadata-ingestion/setup.cfg

```diff
@@ -42,6 +42,8 @@ ignore_missing_imports = yes
 ignore_missing_imports = yes
 [mypy-snowflake.*]
 ignore_missing_imports = yes
+[mypy-druid.*]
```

Comment:
I think you'd want `[mypy-pydruid.*]`, since the actual python package is called `pydruid`